### PR TITLE
Python 3 Migration

### PR DESCRIPTION
**Pull request by MiltonLn**

### CODE REVIEW

**Review by skqr — COMMENTED** [review]
Looks great - maybe increase the version in setup.py so that we can release as soon as merged?

File: pyrovider/tools/dicttools.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def dictpath(dictionary: dict, path: list):
 
 def dictiter(arg):
     if isinstance(arg, dict):
-        return iter(arg.items())
+        return iter(list(arg.items()))
```
Comment (tutuca) [inline pyrovider/tools/dicttools.py]: `arg.items` is already an iterator, remove both `iter` and `list` castings.

Comment (MiltonLn) [reply pyrovider/tools/dicttools.py]: `2to3` :sigh:

**Review by skqr — APPROVED** [review]
Awesome!

**Pull request merged**